sem/runner.py

```diff
@@ -97,6 +97,10 @@ def __init__(self, path, script, optimized=True):
         self.script_executable = max(match_percentages,
                                      key=lambda x: x['percentage'])['path']
 
+        if optimized and "scratch" in self.script_executable:
```

Comment:
Are scripts in ns-3-dev/examples/ and ns-3-dev/src/<model-name>/examples be supported?

Reply:
Yes! Only those in scratch gave some trouble, because of a wrong path in build-status.py